scripts/submit_eval_jobs.py

```diff
@@ -39,110 +49,16 @@
     "alpaca_eval",
 ]
 
-# model to evaluate, each in the followng format: model name, their beaker id, checkpoint subfolder
+# format: model name, their beaker id, checkpoint subfolder, tuned or base.
+# or: name, path, None, tuned or base
 models = [
```

Comment:
Can we move these models into a spreadsheet or somewhere to make them easily sharable?

Reply:
Yup, this is happening: https://beaker.org/ws/ai2/adapt-models/datasets?sort=created:descending&results=false&committed=true
@dwadden is thinking about how to share this better too.